doc/develop/fixing_data.rst

```diff
@@ -354,6 +365,40 @@ For example for monthly data, place the files in the ``/Tier3/MSWEP/latestversio
 
 For more info: http://www.gloh2o.org/
 
+.. _fixing_native_models:
+
+Native models : IPSL-CM6,...
+-----------------------------
+
+The following models are natively supported through the procedure
+described above (:ref:`fix_structure`) and at
+:ref:`configure_native_models`:
+
+  - **IPSL-CM6** : both output formats (i.e. the ``Output`` and the
```

Comment:
I suggest to move to a sub-section style approach also for the models. In addition to the following suggestion, this requires a change of indentation of the whole IPSL-CM6 part.

```suggestion
IPSL-CM6
~~~~~~~~
Both output formats (i.e. the ``Output`` and the
```